service/src/main/kotlin/io/provenance/api/frameworks/web/external/provenance/ProvenanceApi.kt

```diff
@@ -148,11 +149,40 @@ class ProvenanceApi {
                     ApiResponse(
                         responseCode = "200",
                         description = "successful operation",
-                        content = [Content(schema = Schema(implementation = ScopeResponse::class))]
+                        content = [Content(schema = Schema(implementation = QueryScopeResponse::class))]
```

Comment:
I think this was just a typo @cworsnop-figure, but let me know if I'm missing something.